library/src/main/java/com/novoda/downloadmanager/lib/DownloadsRepository.java

```diff
@@ -195,6 +198,71 @@ public void setDownloadSubmitted(FileDownloadInfo info) {
         contentResolver.update(info.getAllDownloadsUri(), contentValues, null, null);
     }
 
+    @Nullable
+    String getCurrentDownloadingBatchId() {
+        return getCurrentBatchIdWithColumnStatus(DownloadStatus.RUNNING);
```

Comment:
Correct, we only support sequential downloads for now.

In case that we want to make parallel downloads in the future this would return an array